Paper.tex

```diff
@@ -1422,15 +1367,11 @@ \subsection{State \& Nonce Validation}\label{sec:statenoncevalidation}
 
 \hypertarget{Phi}{}And finally we define $\Phi$, the block transition function, which maps an incomplete block $B$ to a complete block $B'$:
 \begin{eqnarray}
-\Phi(B) & \equiv & B': \quad B' = B^* \quad \text{except:} \\
-B'_{\mathrm{n}} & = & n: \quad x \leqslant \frac{2^{256}}{\hyperlink{H__d}{H_{\mathrm{d}}}} \\
-B'_{\mathrm{m}} & = & m \quad \text{with } (x, m) = \mathtt{PoW}(B^*_{\hyperlink{H_cancel_n}{\hcancel{n}}}, n, \mathbf{d}) \\
-B^* & \equiv & B \quad \text{except:} \quad {\hyperlink{Transaction_Receipt}{B^*_{\mathrm{r}}}} = \hyperlink{r}{r}(\hyperlink{Pi}{\Pi}(\Gamma(B), B))
+\Phi(B) & \equiv & B': \quad B' = B \quad \text{except:} \\
+{\hyperlink{Transaction_Receipt}{{B'_{\mathrm{H}}}_{\mathrm{r}}}} & = & \hyperlink{r}{r}(\hyperlink{Pi}{\Pi}(\Gamma(B), B))
```

Comment:
I think B_r was referring to the state root header property here, so this change would make that more clear, though I'm still a bit confused by this line.

Reply:
Yes, I think that `H_r` refers to the state root. The hyperlinks look wrong here. And I think it should be `TRIE(L_S(...))` instead of `r(...)`, like in Eq (33) or (173). I suggest to change to
```
{B'_{\mathrm{H}}}_{\mathrm{r}} & = & \mathtt{TRIE}(L_{\mathrm{S}}(\hyperlink{Pi}{\Pi}(\Gamma(B), B)))
```

Reply:
I agree, and I made the change